p2p/p2p.go

```diff
@@ -331,3 +331,18 @@ type ConnectionsWatcher interface {
 	Close() error
 	IsInterfaceNil() bool
 }
+
+// PeersRatingHandler represent an entity able to handle peers ratings
+type PeersRatingHandler interface {
+	AddPeer(pid core.PeerID)
+	IncreaseRating(pid core.PeerID)
+	DecreaseRating(pid core.PeerID)
+	GetTopRatedPeersFromList(peers []core.PeerID, numOfPeers int) []core.PeerID
+	IsInterfaceNil() bool
+}
+
+// IntRandomizer interface provides functionality over generating integer numbers
+type IntRandomizer interface {
```

Comment:
why do we need to define this interface in the `p2p` package

Reply:
removed it